task_processing/plugins/mesos/logging_executor.py

```diff
@@ -34,9 +35,21 @@ class LogMetadata(PRecord):
     executor_id = field(type=str, initial='')
 
 
+def standard_handler(task_id, message, stream):
+    print(message, file=sys.stderr if stream is 'stderr' else sys.stdout)
+
+
 class MesosLoggingExecutor(TaskExecutor):
-    def __init__(self, downstream_executor):
+    def __init__(
+        self,
+        downstream_executor,
+        handler=standard_handler,
+        format_string=DEFAULT_FORMAT,
+    ):
         self.downstream_executor = downstream_executor
+        self.TASK_CONFIG_INTERFACE = downstream_executor.TASK_CONFIG_INTERFACE
```

Comment:
is this here so that the executor satisfies the interface?

Reply:
Yeah, I put this in because I'm getting the task interface from the runner: https://github.com/Yelp/Tron/blob/master/tron/mesos.py#L204

I could save the mesos task config interface instead of getting it from the runner. But this way, the stack's TASK_CONFIG_INTERFACE doesn't change when you add the logging executor, which I think makes more sense?